test/scripts/apidoc/signature.example.ts

```diff
@@ -1,4 +1,5 @@
 import type { Casing, ColorFormat } from '../../../src';
+import { FakerError } from '../../../src';
```

Comment:
Here in test it doesn't make much difference, but we import `FakerError` from its exact script on all other places as inside the src it can somehow result in bundle errors

```suggestion
import { FakerError } from '../../../src/errors/faker-error';
```